test-dxcompiler-in-project/Makefile

```diff
@@ -1,6 +1,6 @@
 # TODO: find a way to download from draft release URLs; the below works for final releases.
```

Comment:
With this setup, we could remove the test-dxcompiler-in-project applet, since it wouldn't be used.

Reply:
correct